doc/help/elektra-cmerge-strategy.md

```diff
@@ -0,0 +1,32 @@
+# elektra-cmerge-strategies(7) -- how to merge key sets
+
+## 3-WAY
+
```

Comment:
Also here some introduction text would be helpful. What is this text about? (terminology)